Userland/Libraries/LibGfx/Rect.h

```diff
@@ -422,6 +422,11 @@ class Rect {
 
     void intersect(const Rect<T>&);
 
+    static Rect<T> centered(const Point<T>& center, const Size<T>& size)
```

Comment:
`centered_on()`? I think there's a `center_within()` method, so the generic sounding name might be confusing